MapleServer2/Packets/MasteryPacket.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Enums;
+
+namespace MapleServer2.Packets
+{
+    public static class MasteryPacket
+    {
+        public static Packet SetMasteryExp(byte masteryType, long totalMasteryExp)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.MASTERY);
+
+            pWriter.WriteByte();
+            pWriter.WriteEnum((MasteryType) masteryType);
+            pWriter.WriteLong(totalMasteryExp);
+
+            return pWriter;
+        }
+        
+        public static Packet ClaimReward(int rewardBoxDetails, int quantity, int itemId)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.MASTERY);
+
+            pWriter.WriteByte(0x01);
```

Comment:
Maybe convert this into a "mode", similar to how other packets do this, since this is likely the mode 1 for this, and line 13 is probably mode 0